src/components/Tag.tsx

```diff
@@ -0,0 +1,15 @@
+interface TagProps {
+    name: string;
+    backgroundColor: string;
+}
+
+export default function Tag({ name, backgroundColor }: TagProps) {
+    return (
+        <div
+            className={`h-fit w-fit rounded-xl border border-slate-500 px-3 py-1.5 text-white`}
```

Comment:
Use `border-black` instead of `border-slate-500` and use `border-2` instead of `border`.